assets/js/modules/idea-hub/datastore/module-data.test.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * `modules/idea-hub` data store: module-data tests.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	createTestRegistry,
+	unsubscribeFromAll,
+	untilResolved,
+} from '../../../../../tests/js/utils';
+import { enabledFeatures } from '../../../features';
+import { MODULES_IDEA_HUB } from './constants';
+
+describe( 'modules/idea-hub module-data', () => {
+	const baseInfoVar = '_googlesitekitIdeaHub';
+	const baseInfo = {
```

Comment:
These look like they were copied from the `core/site` tests. Let's rename these to be more specific to this test.
```suggestion
	const ideaHubDataGlobal = '_googlesitekitIdeaHub';
```
As for `baseInfo` here, it looks like it's only used in the one test for selectors, so how about we inline it there instead?